values-tooling.yaml

```diff
@@ -34,5 +34,12 @@ applications:
     source_path: charts/ubiquitous-journey
     source_ref: master
     sync_policy_automated: true
-    valueFiles:
+    helm_values:
     - apps/sonarqube/values.yaml
+  - name: crw
+    enabled: true
+    destination: crw
+    source: https://github.com/eformat/argocd
+    source_path: crw/base
```

Comment:
is this a temporary thing or should we take your CRW stuff out of that project?

Reply:
this is temporary, i will pull it out into rht-labs/<something> project